box.json

```diff
@@ -52,14 +52,13 @@
         "format":"cfformat run system/**/*.cfc,tests/specs/**/*.cfc --overwrite",
         "format:watch":"cfformat watch system/**/*.cfc,tests/specs/**/*.cfc ./.cfformat.json",
         "format:check":"cfformat check system/**/*.cfc,tests/specs/**/*.cfc",
+        "log":"server log --follow",
+        "onServerInstall":"cfpm install zip,orm,mysql,postgresql,sqlserver,document,feed,mail,debugger",
```

Comment:
this won't work, because `cfpm` looks for the server in the default location and not the overriden one we use.  Plus this will exdcute for ALL servers, and it should only do it for adobe 2021

Reply:
@lmajano This is actually changed now.  
* cfpm command will ignore any Lucee servers and will not error
* cfpm will search specifically for an ACF 2021+ server and ignore others
* cfpm when run as part of any server-related interception point will be smart enough to apply itself to the specific server that fired the interception.  

I had Michael change this to greatly simplify the setup.